importer/Import/Repodata.hs

```diff
@@ -94,7 +94,11 @@ loadRepoFromURI uri = do
 
  where
     appendOrThrow :: T.Text -> URI
-    appendOrThrow path = appendURI uri (T.unpack path) `throwIfNothing` RepoException
+    appendOrThrow path = let
+          -- If the URI path already ends in a slash, don't add an extra one
+          new_path = if last (uriPath uri) == '/' then T.unpack path else "/" ++ T.unpack path
```

Comment:
Btw to use `T.isSuffixOf` we also need to convert from String to Text using `T.pack` like so:
```
if "/" `T.isSuffixOf` T.pack (uriPath uri)
```

which I think is a bit less efficient.